lib/galaxy/web/base/interactive_environments.py

```diff
@@ -219,7 +219,9 @@ def launch(self, raw_cmd=None, env_override={}, volumes=[]):
             return None
         else:
             log.debug( "Container id: %s" % stdout)
-            port_mappings = self.get_proxied_ports(stdout)
+            port_mappings, gateway_ip = self.get_proxied_ports(stdout)
+            if self.attr.docker_hostname == 'localhost':
```

Comment:
I think I'm ok with this, since it only affects situations in which `docker_hostname` is localhost, so none of the ones that @bgruening or @natefoo encounter where docker is isolated on another host, but I'd rather they chime in since it affects them as well.